airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/AirbyteSchemaIdentityMapper.kt

```diff
@@ -4,10 +4,13 @@
 
 package io.airbyte.cdk.load.data
 
-interface AirbyteSchemaIdentityMapper {
-    fun map(schema: AirbyteType): AirbyteType =
+interface AirbyteSchemaMapper {
```

Comment:
This leaked in from the next change. But the point is to make the mappers generalizable for chaining

Reply:
is the idea that we only strictly _need_ the `map` method for chaining, and all the mapXyz methods are just for convenience?

Reply:
yeah, I think `IdentityMapper` is the wrong name, it's more like TypeAwareMapper or something. So `Mapper => just `map`. The IdentityMapper class has all the bells and whistles.

In practical terms, I needed a common parent type for the Noop and visiting value mappers, so I broke out an interface for both of them for symmetry